spanner/api/Spanner.Samples.Tests/GetDatabaseDdlAsyncTest.cs

```diff
@@ -48,9 +48,9 @@ public async Task TestGetDatabaseDdlAsync()
         await _spannerFixture.RunWithTemporaryDatabaseAsync(_spannerFixture.InstanceIdWithMultiRegion, databaseId, async databaseId =>
         {
             var sample = new GetDatabaseDdlAsyncSample();
-            var statements =
+            var response =
                 await sample.GetDatabaseDdlAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceIdWithMultiRegion, databaseId);
-            Assert.Collection(statements,
+            Assert.Collection(response.Statements,
```

Comment:
I don't really understand how this change works - I'd have expected it to fail before. But hey, if it passes, that's fine.

Reply:
The sample method was previously returning the statements only, I made it return the response object.